.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+**What operating system does the project run on?**
```

Comment:
"and version"?

Perhaps: "Which operating system and version is the project developed on?" is actually what we need?